docs/proposals/PROPOSAL_F.md

```diff
@@ -0,0 +1,374 @@
+         "size": 528,
+         "platform": {
+            "architecture": "amd64",
+            "os": "linux"
+         },
+         // signatures added to the linux/amd64
+         "annotations": {
+            "dev.cosignproject.cosign/signature.e59879c": "...",
+            "dev.sigstore.cosign/bundle.6436749": "...",
+            "dev.sigstore.cosign/certificate.776a3f1": "...",
+            "dev.sigstore.cosign/chain.8adf60e": "...",
+            "dev.cosignproject.cosign/signature.de25bf6": "...",
+            "dev.sigstore.cosign/bundle.37d34ff": "...",
+            "dev.sigstore.cosign/certificate.e8adf60": "...",
+            "dev.sigstore.cosign/chain.b1c64a3": "...",
+            "org.opencontainers.signatures.index":"dev.cosignproject.cosign/signature.e59879c,dev.cosignproject.cosign/signature.de25bf6,dev.sigstore.cosign/bundle.37d34ff,dev.sigstore.cosign/certificate.e8adf60,dev.sigstore.cosign/chain.b1c64a3,dev.sigstore.cosign/bundle.6436749,dev.sigstore.cosign/certificate.776a3f1,dev.sigstore.cosign/chain.8adf60e"
+         }
+      },
+      // artifacts referencing the image manifests
+      {
+         "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
+         "digest": "sha256:5b0044a1244...", // arm64 SBOM
+         "size": 1024,
+         "annotations": {
+            "org.opencontainers.reference.type": "sbom",
+            "org.opencontainers.reference.digest": "sha256:c3c58223e2af75154c4a7852d6924b4cc51a00c821553bbd9b3319481131b2e0",
+            "org.opencontainers.reference.description": "Software Bill of Materials of alpine:3.17 linux/arm64 image"
+         }
+      },
+      {
+         "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
+         "digest": "sha256:5b0044a44d...", // amd64 SBOM
+         "size": 1024,
+         "annotations": {
+            "org.opencontainers.reference.type": "sbom",
+            "org.opencontainers.reference.digest": "sha256:4ff3ca91275773af45cb4b0834e12b7eb47d1c18f770a0b151381cd227f4c253",
+            "org.opencontainers.reference.description": "Software Bill of Materials of alpine:3.17 linux/amd64 image"
+         }
+      }
+   ]
+}
+```
+
+#### Enhanced image
+
+An enhanced image can be pushed directly, with all metadata annotations and references.
+There is no need to nest index in this scenario.
+
+```json
+{
+   "mediaType": "application/vnd.oci.image.index.v1+json",
+   "schemaVersion": 2,
+   "manifests": [
+      // Image manifests with signature annotations
+      {
+         "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
+         "digest": "sha256:c3c58223e2af75154c4a7852d6924b4cc51a00c821553bbd9b3319481131b2e0",
+         "size": 528,
+         "platform": {
+            "architecture": "arm64",
+            "os": "linux"
+         },
+         // signatures added to the linux/arm64
+         "annotations": {
+            "dev.cosignproject.cosign/signature.e59879c": "...",
```

Comment:
Just to clarify, the working group [mission statement](https://github.com/opencontainers/wg-reference-types#mission-statement) is:
> Propose how to describe and query relationships between objects stored in an OCI registry.

@sudo-bmitch @nishakm @jdolitsky @imjasonh I believe that exactly how images are signed or SBOMs are stored is out of scope for this working group. Is that correct?

I raise this as I think we should be clear that we're using illustrative examples to solve for the working group's mission, not trying to define signing or SBOM for artifacts across the industry.

Reply:
Correct, the goal is to define how additional data is associated to an existing image. We don't want to get into signing formats. It might make sense to discuss this not as signatures, but an optimization for smaller data types.

Reply:
Yes, I tried to keep the proposal as generic as I could, those signatures only appear in the examples. Changing them with what @imjasonh suggested 👍

Reply:
Fixed https://github.com/opencontainers/wg-reference-types/pull/52/commits/47a0daa6a00ac31c25ba0d180c158ff9442e6634